utility/src/main/java/org/oppia/android/util/logging/EventBundleCreator.kt

```diff
@@ -84,6 +105,43 @@ class EventBundleCreator @Inject constructor(
     }?.activityName ?: "unknown_activity_context"
   }
 
+  fun fillPerformanceMetricsEventBundle(oppiaMetricLog: OppiaMetricLog, bundle: Bundle): String {
+    bundle.putLong("timestamp", oppiaMetricLog.timestampMillis)
+    bundle.putString("priority", oppiaMetricLog.priority.toAnalyticsName())
+    bundle.putBoolean("is_app_in_foreground", oppiaMetricLog.isAppInForeground)
+    bundle.putString("memory_tier", oppiaMetricLog.memoryTier.toAnalyticsName())
+    bundle.putString("storage_tier", oppiaMetricLog.storageTier.toAnalyticsName())
+    bundle.putString("network_type", oppiaMetricLog.networkType.toAnalyticsName())
+    bundle.putString("current_screen", oppiaMetricLog.currentScreen.toAnalyticsName())
+    return oppiaMetricLog.loggableMetric.convertToLoggableMetricType()?.also { loggableMetric ->
+      // No allowance for user IDs to be logged since learner study feature is disabled for
+      // performance metrics logging as of now.
+      loggableMetric.storeValue(PropertyStore(bundle, allowUserIds = false))
+    }?.metricName ?: "unknown_loggable_metric"
+  }
+
+  private fun OppiaMetricLog.LoggableMetric.convertToLoggableMetricType():
```

Comment:
A nit on member ordering: since the method above appears after the other public API method in this class, I feel like this helper should appear after its helper (since it appears next in the class order).